package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "tezster-gui-v2.0",
+  "name": "tezster-gui",
   "productName": "Tezster-GUI",
-  "version": "2.0.0",
-  "description": "Tezster-GUI",
+  "version": "2.1.0",
+  "description": "GUI version of Tezster which is ready to use Desktop app, made with React and Electron",
   "scripts": {
     "build": "concurrently \"yarn build-main\" \"yarn build-renderer\"",
```

Comment:
change 'appId' for electron to a relevant appId.

Reply:
what shall I put as appId can you please suggest

Reply:
"appId": "com.electron.tezster-gui" I have used as an app id